### PR TITLE
Portfolio mobile version progress

### PR DESCRIPTION
**Pull request by alexisuwimana**
this are changes to comple the project

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -45,13 +45,14 @@ <h2 class="heading-2">I am Software Developer</h2>
...
     <section class="Works">
         <div class="works-title" >
            <h3>My Recent Works </h3> 
            <hr class="works-line">
         </div>
         <div class="works-img">
-            <img src="./images/works/img-works-0.svg">
+            <img src="./images/works/img-works-0.svg" alt="works">
         </div>
         <div class="stories">
```
Comment (uwadonat) [inline index.html:57]: - [ ] Please use  background images instead of img tags

File: styles.css [hunk 2]
```diff
@@ -261,21 +263,21 @@ hr.works-line {
...
-  margin: 12px 0px;
+  margin: 12px 0.01px;
   margin: auto;
 }
 
 .stories-lang-list {
   /* Auto Layout */
-  
+
   display: grid;
```
Comment (uwadonat) [inline styles.css:289]: - [ ] kindly use the font,  margins (top, bottom, left, right) defined in the design. It is best practice to pay attention to the details given.
- your design
Screenshot from 2021-09-25 07-34-26
-Figma design
Screenshot from 2021-09-25 07-47-01

**Review by uwadonat — CHANGES_REQUESTED** [review]
Hi **@alexisuwimana**, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: styles.css [hunk 3]
```diff
@@ -221,25 +222,30 @@ li {
...
   font-style: normal;
   font-weight: normal;
-  font-size: 40px;
+  font-size: 32px;
   line-height: 52px;
-  display: flex;
-  align-items: center;
+  color: #1c1a19;
+
+  /* background-image: url(./images/works/img-works-0.svg);
```
Comment (carlos-ssh) [inline styles.css]: [OPTIONAL]
**Keep good practices and professionalism.**
- Remove commented code.

File: styles.css [hunk 4]
```diff
@@ -221,25 +222,30 @@ li {
...
   background: #fdfdfd;
   position: absolute;
   width: 100%;
   height: 3658.4px;
-  left: 0px;top: 682px;
-  font-family: Crete Round;
+  left: 0.001px;
+  top: 682px;
+
+  /* decoration */
```
Comment (carlos-ssh) [inline styles.css]: [OPTIONAL]
Your code explains by itself what it do.
Comments are helpful on a development environment to provide support to your partner.
Once that you are requesting a code review it's recommended to have a clean code without comments at least linters can keep.

**Review by carlos-ssh — APPROVED** [review]
Hi @alexisuwimana, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**